modules/document_repository/ajax/documentDelete.php

```diff
@@ -67,4 +72,18 @@
     unlink($path);
 }
 
+// Cleanup empty directories
+set_error_handler(
+    function () {
+        // Silence the E_WARNING when files exist in the directory.
+    }
+);
+$rm_directory = __DIR__ . '/../user_uploads/'
+    . substr($dataDir, 0, strlen($dataDir)-(strlen($fileName)+1));
```

Comment:
Are you trying to chop off the filename from `$dataDir` here? If so, just use [`dirname()`](https://secure.php.net/manual/en/function.dirname.php);